packages/maskbook/src/components/DataSource/useMyPersonas.ts

```diff
@@ -19,7 +18,10 @@ const independentRef = {
             Services.Identity.queryMyPersonas().then((p) => {
                 independentRef.myPersonasRef.value = p.filter((x) => !x.uninitialized)
                 independentRef.myUninitializedPersonasRef.value = p.filter((x) => x.uninitialized)
-                setStorage<boolean>('mobileIsMyPersonasInitialized', independentRef.myPersonasRef.value.length > 0)
+                Services.Storage.setStorage<boolean>(
```

Comment:
Can you add a new function in `SettingsService` instead of expose this Storage service directly?

Reply:
Included storage into `HelperServices`.